src/apps/chifra/pkg/output/json.go

```diff
@@ -20,29 +20,44 @@ import (
 	"github.com/TrueBlocks/trueblocks-core/src/apps/chifra/pkg/validate"
 )
 
-// PrintJson marshals its arguments and prints JSON in a standardized format
-func PrintJson(serializable interface{}) error {
-	var response map[string]interface{}
+type JsonFormatted struct {
+	Data   interface{} `json:"data,omitempty"`
+	Errors []string    `json:"errors,omitempty"`
+	Meta   *Meta       `json:"meta,omitempty"`
+}
 
-	if Format == "json" {
-		response = map[string]interface{}{
-			"data": serializable,
-		}
+// AsJsonBytes marshals JsonFormatted struct, populating Meta field if
+// needed
+func AsJsonBytes(j *JsonFormatted) ([]byte, error) {
+	var result JsonFormatted
 
+	if Format == "json" {
+		result = *j
 	} else {
+		// TODO: global validate.Errors is not server-safe
```

Comment:
Can you explain this to me? I'm sure it's true -- globals are almost always bad -- but how do you know?

Reply:
Let's say there's an error in this array. It was caused by client A connecting and doing something wrong. Then client B connects, new goroutine starts and takes care of B's request. But there is still error in the array, so B will get it even tough they didn't do anything wrong. 
I know that TrueBlocks are a local application, but these two clients could be user opening two tabs in their browser (or maybe our future Electron UI). Or someone using API.

Reply:
Got it completely. Thanks.